test.js

```diff
@@ -47,3 +53,16 @@ test('rule: caret-deps', (t) => {
         }
     });
 });
+
+test('git helper module', async (t) => {
+    const cwd = await mkdirtemp();
+    const { stdout } = exec(`ls ${cwd}`);
```

Comment:
`ls` wouldn't work on Windows. You can accomplish the same thing in a cross-platform manner with the `fs` module.

Reply:
Have to read more about `fs` module deeper